# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: target/desktop/etc_profile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We want a colored prompt as well as a "nicer" history handling.

  - Rene Rebe <rene@exactcode.de>

--- generic_etc_profile	2006-07-28 17:56:19.000000000 +0000
+++ desktop_etc_profile	2006-07-28 17:57:02.000000000 +0000
@@ -43,7 +43,8 @@
 #
 if [ "$PS1" ] ; then
 	PS1='$ ' ; PS2='> ' ; PS4='+ '
-	[ "$BASH_VERSION" ] && PS1='\u@\h:\w\$ '
+	color="1;32m" ; [ "$UID" = 0 ] && color="0;31m"
+	[ "$BASH_VERSION" ] && PS1="\[\033[$color\]\u@\h:\w\\$\[\033[0m\] "
 	[ "$KSH_VERSION" -a   -w / ] && PS1='$USER:$PWD# '
 	[ "$KSH_VERSION" -a ! -w / ] && PS1='$USER:$PWD# '
 fi
@@ -92,6 +93,10 @@
 	alias rehash="hash -r"
 fi
 
+# desktop target specifica
+#
+HISTCONTROL=erasedups
+HISTSIZE=4000; HISTFILESIZE=2000
+
 # include local profiles
 #
 for x in /etc/profile.d/* /etc/conf/profile ; do
